frappe/patches/v15_0/migrate_role_profile_to_table_multi_select.py

```diff
@@ -0,0 +1,13 @@
+import frappe
+
+
+def execute():
+	user = frappe.qb.DocType("User")
+	role_profiles = frappe.qb.from_(user).select(user.name, user.role_profile_name).run(as_dict=True)
+	for user in role_profiles:
+		if not user.role_profile_name:
+			continue
+		user_role_profile = [{"role_profile": user.role_profile_name}]
+		user = frappe.get_doc("User", user.name)
+		user.update({"role_profiles": user_role_profile})
+		user.save()
```

Comment:
1. This will be super slow with a large number of users. Let's just insert a child table row for the role profile and not resave the parent document.
2. This will either require a periodic commit or `bulk_insert` to avoid a massive long-running DB transaction.

Reply:
@niraj2477 bulk insert is simpler IMO. 

https://github.com/frappe/frappe/blob/1b8e9c3502b702178f777cf2e5c253ccec573e2e/frappe/geo/doctype/country/country.py#L31-L34